Mathlib/Topology/UniformSpace/Cauchy.lean

```diff
@@ -54,6 +54,10 @@ theorem cauchy_iff {f : Filter α} : Cauchy f ↔ NeBot f ∧ ∀ s ∈ 𝓤 α,
     simp only [subset_def, Prod.forall, mem_prod_eq, and_imp, id, ball_mem_comm]
 #align cauchy_iff cauchy_iff
 
+lemma cauchy_of_neBot {l : Filter α} [hl : l.NeBot] :
```

Comment:
I'm not sure this is the right name. Is `cauchy_iff_le_of_neBot` too long? Or simply `cauchy_iff_le` since we often don't mention type class assumptions.